### PR TITLE
votca: fix git url and allow build without xtp parts

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

**Comment by junghans** [comment]
@aweits can you review this?

**Comment by junghans** [comment]
@spackbot re-run pipeline

File: var/spack/repos/builtin/packages/votca/package.py [hunk 1]
```diff
@@ -34,12 +35,12 @@ class Votca(CMakePackage):
...
+    depends_on("libxc", when='+xtp')
     depends_on("hdf5+cxx~mpi")
-    depends_on("libint@2.6.0:")
-    depends_on("libecpint")
-    depends_on("py-h5py")
-    depends_on("py-lxml")
+    depends_on("libint@2.6.0:", when='+xtp')
+    depends_on("libecpint", when='+xtp')
+    depends_on("py-h5py", when='+xtp')
+    depends_on("py-lxml", when='+xtp')
```
Comment (tgamblin) [inline var/spack/repos/builtin/packages/votca/package.py]: Can you put all the xtp dependencies inside a `with when("+xtp"):` block?

**Review by tgamblin — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/votca/package.py (hunk 1 above)
Comment (junghans) [reply var/spack/repos/builtin/packages/votca/package.py]: Done

**Comment by junghans** [comment]
@tgamblin can you review this again?

**Review by tgamblin — APPROVED** [review]

**Pull request merged**